packages/devtools_app/lib/src/debugger/codeview.dart

```diff
@@ -402,15 +402,18 @@ class _CodeViewState extends State<CodeView>
     );
   }
 
-  Widget wrapInElevatedCard(Widget widget) {
+  Widget wrapInElevatedCard(
+    Widget widget, {
+    double width = wideSearchTextWidth,
```

Comment:
this extra parameter is never passed in anywhere so should be removed.

Reply:
Which extra parameter? The `width` parameter is being used on line 429 below